packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -80,9 +81,8 @@ export class ExpandableSection extends React.Component<ExpandableSectionProps, E
     // uncontrolled
     if (isExpanded === undefined) {
       propOrStateIsExpanded = this.state.isExpanded;
-      onToggle = () => {
-        onToggleProp();
-        this.setState(prevState => ({ isExpanded: !prevState.isExpanded }));
+      onToggle = isExpanded => {
```

Comment:
We need to account for the case when isExpanded is not provided.

Reply:
`isExpanded` here is _always_ provided because on line `110` is where this function is called and it supplies a boolean param.

Reply:
@boaz @christianvogt I know this is a day off for some I can fix linting error so we can this into the release.